skyportal/tests/api/test_tag.py

```diff
@@ -0,0 +1,135 @@
+import uuid
+
+import pytest
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from skyportal.models import ObjTagOption
+from skyportal.tests import api
+
+
+# --- Testing ObjTagOption API
+def test_get_tag(super_admin_token):
+    tag_to_create = {"name": f"TestTag{uuid.uuid4().hex}"}
+    status, _ = api("POST", "objtagoption", data=tag_to_create, token=super_admin_token)
```

Comment:
maybe for at least one combination of POST/GET/PATCH/DELETE + endpoint + status) , you might want to check that the content of `data` is valid too, not just the status? doesn't have to be for everything that's really the cherry on top, but at least for the GETs, you should really really check that the content if correct, because well if you get a 200 but the list is empty when it shouldn't, that's an issue! Especially as we add data access and want to test who can see what, that will be absolutely necessary,